File: src/my_project/easy_problems/from151To200/goat_latin.py
class Solution:
    def toGoatLatin(self, sentence: str) -> str:
        vowels = ['a','e','i','o','u', 'A','E','I','O','U']
        list_vowels = sentence.split(' ')
        print(list_vowels)

        for i in range(len(list_vowels)):
            if list_vowels[i][0] in vowels:
                list_vowels[i] += 'ma' + (i+1)*'a'
            else: 
                list_vowels[i] = list_vowels[i][1:] + list_vowels[i][0] + 'ma' + (i+1)*'a'

        

            
            

        return ' '.join(list_vowels)


solution = Solution()

print(solution.toGoatLatin(sentence = "I speak Goat Latin"))

File: my_project_typescript/easy_problems_typescript/from51To100/goatLatin.ts
function toGoatLatin(sentence: string): string {

    let vowels: Array<string> = ['a','e','i','o','u', 'A','E','I','O','U']
    let listVowels: Array<string> = sentence.split(' ')

    for (let i = 0; i < listVowels.length; i++){
        if (vowels.includes(listVowels[i][0])){
            listVowels[i] += 'ma' + 'a'.repeat(i+1)
        }
        else {
            listVowels[i] = listVowels[i].slice(1) + listVowels[i][0] + 'ma' + 'a'.repeat(i+1)
        }
    }

    return listVowels.join(' ')

};


console.log(toGoatLatin("I speak Goat Latin"))